<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Prathyush-PC" ignored="0" name="test 4" tests="1" failures="1" timestamp="30 Jan 2019 14:28:10 GMT" time="15.514" errors="0">
  <testcase name="IAPV2_002" time="15.452" classname="com.wibmo.IAPV2.tests.IAPV2_002">
    <failure type="java.lang.AssertionError" message="expected [080] but found [070]">
      <![CDATA[java.lang.AssertionError: expected [080] but found [070]
	at com.wibmo.IAPV2.tests.IAPV2_002.IAPV2_002(IAPV2_002.java:61)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- IAPV2_002 -->
</testsuite> <!-- test 4 -->
